tooling/inspector/src/main.rs

```diff
@@ -0,0 +1,77 @@
+use std::{path::PathBuf, process::exit};
+
+use acir::circuit::ExpressionWidth;
+use clap::Parser;
+use noirc_artifacts::program::ProgramArtifact;
+use noirc_artifacts_info::{count_opcodes_and_gates_in_program, show_info_report, InfoReport};
+
+#[derive(Debug, Clone, Parser)]
+pub(crate) struct Args {
+    /// The artifact to inspect
+    artifact: PathBuf,
+
+    /// Output a JSON formatted report. Changes to this format are not currently considered breaking.
+    #[clap(long, hide = true)]
+    json: bool,
+
+    /// Specify the backend expression width that should be assumed
+    #[arg(long, value_parser = parse_expression_width)]
+    expression_width: Option<ExpressionWidth>,
+
+    /// Display the ACIR for compiled circuit
+    #[arg(long)]
+    print_acir: bool,
+}
+
+pub const DEFAULT_EXPRESSION_WIDTH: ExpressionWidth = ExpressionWidth::Bounded { width: 4 };
```

Comment:
`DEFAULT_EXPRESSION_WIDTH` is in `noirc_driver` but I didn't want to depend on that, and I wasn't sure if it was okay to move it to the same crate `ExpressionWidth` is.

Reply:
I'm inclined to omit this information if we're relying on the user to provide it for us.

Reply:
In the end I found that the expression width was only used to print it in the final report... so I made it optional (in the code) and now there's no command line argument for it.